docs/release/cut-sw-release.md

```diff
@@ -318,9 +266,17 @@ The following instructions are meant for the release manager (or interim release
 
         We welcome feedback on this release!
 
-2.  The release manager emails the announcement to `vdt-discuss@opensciencegrid.org`
-3.  The release manager asks the GOC to distribute the announcement by [opening a ticket](https://ticket.opensciencegrid.org/goc/other)
-4.  The release manager closes the tickets marked 'Ready for Release' in the release's JIRA filter using the 'bulk change' function.
+2.  The release manager uses the [osg-notify tool](https://opensciencegrid.org/operations/services/sending-announcements/) to send the release announcement using the following command:
```

Comment:
As discussed, let's change this line to just say that the following command needs to be run from `submit-1`? Also, do our external S&R members have access to this host?

Reply:
The release manager or his designated rep does this. I think it is not difficult to get an account on submit-1.chtc.wisc.edu.

Reply:
submit-1 has the prerequisites. You still have to setup the git repository as described in the linked document.

Reply:
submit-1 has the *software* prerequisites, it is *not* in the DNS SPF record.

Reply:
@djw8605 : When will the SPF record be updated?

Reply:
Hi, sorry I didn't get this notification.  Anyways, do you have an ip range for the DNS SPF record.  I can add it to the DNS entry then.

Reply:
@timtheisen please update the line now that the SPF record has been updated